node/src/command.rs

```diff
@@ -436,6 +446,37 @@ pub fn run() -> Result<()> {
 
             Ok(())
         }
+        #[cfg(feature = "try-runtime")]
```

Comment:
If we only want to build this branch on environments that are not under `test`: `#[cfg(all(not(test), feature = "try-runtime"))]. This might help with reducing overall build time per pull request.

Reply:
As discussed on Slack, it would be a bit tricky to generalize it.